src/stan/lang/ast/node/cholesky_factor_var_decl_def.hpp

```diff
@@ -14,10 +14,10 @@ namespace stan {
     cholesky_factor_var_decl::cholesky_factor_var_decl(expression const& M,
                                        expression const& N,
                                        std::string const& name,
-                                       std::vector<expression> const& dims)
```

Comment:
this one has it for all the arguments;  sorry that this was all my fault doing it inconsistently to begin with.